### ISSUE TITLE
PowerCommands "Format document on save" and "Remove and Sort Usings on save" conflict with Fix Mixed Tabs

### ISSUE THREAD

**Issue by SvenAelterman** [Microsoft/VS-PPT#110]
When "Format Document on save" is on, tabs are replaced with spaces. (Should be this configurable somewhere in document format settings to keep tabs instead of spaces... I don't recall this behavior in VS 2015?)
When "Remove and Sort Usings on save" is on, the leading tab in front of the class definition is replaced with spaces.
In these cases, after saving, the next thing you see is the Fix Mixed Tabs bar on top of the editor asking to fix. When you fix and save, the same issue occurs and you get stuck in this loop.

**Comment by olegtk** [comment]
It should be no different from invoking commands "Format Document" and "Remove and Sort Usings" manually. Do they behave the same?

**Comment by justcla** [comment]
Yes, this is configurable. When formatting, it uses the tabs/spaces settings that you have defined for that file type. Check your settings in Tools->Options (ie. Text Editor -> C# -> Tabs). If you have it set to "Insert spaces", then on Save, all tabs will be replaced by spaces.
